wiki/markdown/subscription-API.md

```diff
@@ -97,3 +97,11 @@ Delete a single specific subscription
 Curl command example
 
     curl -X DELETE -H "Content-type: application/json"  http://<host>:8090/subscriptions/<name>
+    
+   
+ ## Subscription related information in front-end documentation
+ 
+Read more about curl examples for subscription endpoint in Eiffel Intelligence front-end docs [here](https://github.com/eiffel-community/eiffel-intelligence-frontend/blob/master/wiki/markdown/curl-examples.md#subscriptions).
+
```

Comment:
Please add a title for this link as well since previous title states that it links to fron and but this link does not.

Reply:
I removed it as it is not required. next commit